R/a11y_panel.R

```diff
@@ -0,0 +1,250 @@
+      if (all(is.na(specific_issues))) {
+        shiny::tags$p(
+          "This website is fully compliant with the ",
+          external_link(
+            href = "https://www.w3.org/TR/WCAG22/",
+            "Web Content Accessibility Guidelines version 2.2 AA standard"
+          ),
+          "."
+        )
+      } else {
+        shiny::tagList(
+          shiny::tags$p(
+            "This website is partially compliant with the ",
+            external_link(
+              href = "https://www.w3.org/TR/WCAG22/",
+              "Web Content Accessibility Guidelines version 2.2 AA standard"
+            ),
+            " due to the non-compliances listed below."
+          ),
+          shiny::tags$h3("Non accessible content"),
+          shiny::tags$p("The content listed below is non-accessible for the following reasons.
+             We will address these issues to ensure our content is accessible."),
+          shiny::tags$div(tags$ul(
+            tagList(lapply(specific_issues, shiny::tags$li))
+          ))
+        )
+      },
+      shiny::tags$h3("Disproportionate burden"),
+      shiny::tags$p("Not applicable."),
+      shiny::tags$h2("How we tested this website"),
+      shiny::tags$p(
+        "The template used for this website was last tested on",
+        date_template_reviewed,
+        " against ",
+        external_link(
+          href = "https://www.w3.org/TR/WCAG22/",
+          "Accessibility Guidelines WCAG2.2"
+        ),
+        ". The test was carried out by the ",
+        external_link(
+          href = "https://digitalaccessibilitycentre.org/",
+          "Digital accessibility centre (DAC)"
+        ),
+        "."
+      ),
+      shiny::tags$p(
+        "DAC tested a sample of pages to cover the core functionality of the service including:"
+      ),
+      shiny::tags$div(tags$ul(
+        shiny::tags$li("navigation"),
+        shiny::tags$li("interactive dropdown selections"),
+        shiny::tags$li("charts, maps, and tables")
+      )),
+      shiny::tags$p(
+        "This specific website was was last tested on ", date_tested, " against ",
+        external_link(
+          href = "https://www.w3.org/TR/WCAG22/",
+          "Accessibility Guidelines WCAG2.2"
+        ),
+        ". The test was carried out by the ",
+        external_link(
+          href = "https://www.gov.uk/government/organisations/department-for-education",
+          "Department for Education (DfE)"
+        ),
+        "."
+      ),
+      shiny::tags$h2("What we're doing to improve accessibility"),
+      shiny::tags$p(
+        "We plan to continually test the service for accessibility issues, and are working ",
+        "through a prioritised list of issues to resolve."
+      ),
+      shiny::tags$p(
+        if (!is.na(repo_url)) {
+          shiny::tagList(
+            "Our current list of issues to be resolved is available on our ",
+            external_link(
+              href = paste0(repo_url, "/issues"),
```

Comment:
Feel like there's a risk at some point our validation goes out of sync and allows alternative repo URLs or there's some way that just appending /issues breaks? Maybe an additional argument for `issues_url` would be cleaner and more explicit to someone using the function?

Reply:
Actually yeah I've just broken it first time by doing:
```
dfeshiny::a11y_panel(
    date_tested = "08.08.1995",
    date_prepared = "08.08.1996",
    date_reviewed = "08/08/1997",
    dashboard_url = "https://rsconnect/rsc/dfe-published-data-qa/",
    repo_url = "https://github.com/dfe-analytical-services/dfeshiny/issues",
    dashboard_title = "My work of art"
)
```

That gives this output
![image](https://github.com/user-attachments/assets/72f63177-7707-4d97-9e2d-139205b9bcc4)

Reply:
I think a separate argument is probably least effort and most clear?